src/applications/personalization/dashboard/helpers.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import * as Sentry from '@sentry/browser';
 import { isPlainObject } from 'lodash';
-
+import { format } from 'date-fns';
```

Comment:
I think the `renderWidgetDowntimeNotification` helper is broken and will not render correctly. It looks like the props that are being passed were not updated when the alert was changed to use the `va-alert` web component instead of the old react component.

I realize this isn't part of the PR code itself, but figured it is worth mentioning.